packages/sdk/src/modules/Flywheel.ts

```diff
@@ -111,74 +108,111 @@ export function withFlywheel<TBase extends CreateContractsModule = CreateContrac
       return rewardsInfos;
     }
 
+    async getAllFlywheelClaimableRewards(account: string) {
+      const fplSecondary = this.createFusePoolLensSecondary();
+      const [, comptrollers] = await fplSecondary.callStatic.getFlywheelsToClaim(account, {
+        from: account,
+      });
+
+      // get all markets of all pools
+      const allMarkets: string[] = [];
+      const allFlywheels: string[] = [];
+
+      await Promise.all(
+        comptrollers.map(async (comptroller) => {
```

Comment:
let's break down this call, so it is made separately for each pool - so loop for each of the `comptrollers` and call the `this.getClaimableRewardsForMarkets` with `pool.callStatic.getAllMarkets()` for the markets and `pool.callStatic.getRewardsDistributors()` for the flywheels